assets/docs/sayakpaul/vit_b16_classification/1.md

```diff
@@ -13,7 +13,8 @@ Vision Transformer (ViT) fine-tuned on ImageNet-1k.
 
 ## Overview
 
-This model is a Vision Transformer of type B-16 [1, 2] fine-tuned on the ImageNet-1k dataset [3]. 
+This model is a Vision Transformer of type B-16 [1, 2] fine-tuned on the ImageNet-1k dataset [3]. You can find the complete
```

Comment:
(here and elsewhere)
I would already mention here that the model was trained with AugReg on i21k.